### DISCUSSION TITLE
[Question]: Motivation behind spacing scale

### DISCUSSION THREAD

**Discussion by Holayn** [carbon-design-system/carbon#15848] [Help] [answered]
Hello - I am wondering what the motivation was behind switching from t-shirt sizing to a scale approach for spacing values (#2093). What prompted the change and how has the scale approach proven to be better?

Thanks!

**Comment by tay1orjones** [comment, marked as answer]
Hey my understanding is this was done to ensure the spacing scale could grow and change without having to do things like `xxxs` or `2xl`, `3xl`, etc. 

In that way it's definitely been a success as there have been changes since that were easily made. One of which was deprecating the `layout` tokens to instead have a unified spacing scale, https://github.com/carbon-design-system/carbon/pull/8063

**Reply by Holayn** [reply]
Makes sense, thank you!
